src/manifests/network/cilium/README.md

```diff
@@ -14,8 +14,10 @@ helm repo update
 ```bash
 export CILIUM_NAMESPACE=kube-system
 
-helm template cilium cilium/cilium --version 1.12.6 \
+helm template cilium cilium/cilium --version 1.14.4 \
    --namespace $CILIUM_NAMESPACE \
    --set hubble.relay.enabled=true \
    --set hubble.ui.enabled=true > cilium.yaml
 ```
+
+Modify `cluster-pool-ipv4-cidr: "10.244.0.0/16"`
```

Comment:
What's this setting used for? And why this IP?

Reply:
Well, looks like by default cilium comes with 10.0.0.0/8 for this field, so I tried to keep it consitent to the previous versinon.